client/src/components/AddNewList.jsx

```diff
@@ -29,15 +30,26 @@ const useStyles = makeStyles(() => ({
 
 const AddNewList = () => {
   const classes = useStyles();
+  const [dialogIsOpen, setDialogIsOpen] = useState(false);
```

Comment:
let's make the name something more descriptive like `addListDialogOpen`

Reply:
:+1:  Changed it to `newListDialogIsOpen` and `setNewListDialogIsOpen` since I `addListDialogOpen` kind of sounds like a function.